### ISSUE TITLE
Random occurrences of old version - incompatible with the current backend

### ISSUE THREAD

**Issue by pamasse** [zonemaster/zonemaster-gui#22]
An issue encountered during the tests.

Even after updating, zonemater.fr had two versions displayed: 
1. The ancient version (that is website with blue color) and the 
2. The new version (that is website with grey color)

The ancient version had the "Internal server error" issue.

When you access zonemaster.fr, sometimes we saw the ancient version. Then we had to cache explicitly (not just refresh.. but ctrl+shift+r).

Today, when an user of ZM had the issue, he was accessing the ancient version. 

When tested zonemaster.fr from a laptop suing Chrome, i had also the ancient version, even though two weeks earlier, while i tested, i had flushed my cache and got the new version (i.e. the grey website) in the same configuration. The question was - How did it go to the previous versionN

This issue was quite particular, which is not normal on how cache works. I understand from Pierre that he has a solution to the problem.

=======
disable browser caching for index.html 

`<meta http-equiv="Cache-control" content="no-cache, no-store, must-revalidate">
<meta http-equiv="Pragma" content="no-cache">`

**Comment by sandoche2k** [comment]
An issue encountered during the tests.

Even after updating, zonemater.fr had two versions displayed: 
1. The ancient version (that is website with blue color) and the 
2. The new version (that is website with grey color)

The ancient version had the "Internal server error" issue.

When you access zonemaster.fr, sometimes we saw the ancient version. Then we had to cache explicitly (not just refresh.. but ctrl+shift+r).

Today, when an user of ZM had the issue, he was accessing the ancient version. 

When tested zonemaster.fr from a laptop suing Chrome, i had also the ancient version, even though two weeks earlier, while i tested, i had flushed my cache and got the new version (i.e. the grey website) in the same configuration. The question was - How did it go to the previous versionN

This issue was quite particular, which is not normal on how cache works. I understand from Pierre that he has a solution to the problem.

**Comment by pamasse** [comment]
The issue was caused by the use of [Service Worker](https://developer.mozilla.org/fr/docs/Web/API/Service_Worker_API) on the zonemaster.fr instance. 
The primary usage of the Service Worker is to cache some pages. Service Worker needs to be registered on the client browser to be able to cache pages. 

When we update the GUI with the "classical" gui from the zonemaster repo, we didn't unregister the service worker. 
As result, if your browser has service worker register, you see the index.html cached page. 

Based on this [article](https://love2dev.com/blog/how-to-uninstall-a-service-worker/), I was able to send a script to the browser which tries to unregister the service worker.

It seems to work.
